website/settings/defaults.py

```diff
@@ -1832,3 +1832,19 @@ def parent_dir(path):
 }
 
 PREPRINTS_ASSETS = '/static/img/preprints_assets/'
+
+SOCIAL_FIELDS = {
```

Comment:
I think it should be either (1) a class variable on `OSFUser` (`OSFUser.SOCIAL_FIELDS`) or (2) a module-level variable (`osf.models.user.SOCIAL_FIELDS`). 

(1) seems slightly preferable so that it's easy to access whenever you have `OSFUser` imported.

If you're running into circular import errors, you can import OSFUser in the validator function.